src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -125,4 +125,29 @@ public String getNewQuestion(Entity game_entity, DatastoreService datastore) {
         return (update_game_entity.getProperty("quizQuestion")).toString();
     }
 
+    //Gives the user points if they have taken a quiz
+    public Boolean giveUserPoints(Boolean userQuizStatus, Entity currentUser, DatastoreService datastore) {
+        if(currentUser == null) {
+            log.log(Level.SEVERE, "Given a null {0}", currentUser);
+            return null;
+        }
+
+        if (datastore == null) {
+            log.log(Level.SEVERE, "Given a null {0}", datastore);
+            return null;
+        }
+
+        if(userQuizStatus) {
+            Key user_key = currentUser.getKey();
+            datastore.delete(user_key);
```

Comment:
Deleting a user from the datastore and then adding a new one doesn't seem like the best idea here. What if something went wrong in between deleting and adding the user back to the database? We could lose all of their data. 

We should instead try and just update the values as needed.